frontend/src/app/containers/ExperimentPage.js

```diff
@@ -157,12 +157,13 @@ export class ExperimentDetail extends React.Component {
 
   renderLocaleWarning() {
     const { experiment, locale } = this.props;
-    if (experiment.locales && locale && !experiment.locales.includes(locale)) {
+    if (locale && ((experiment.locales && !experiment.locales.includes(locale)) || (experiment.locale_blocklist && experiment.locale_blocklist.includes(locale)))) {
```

Comment:
We can just land this, it's been sitting long enough. It's minor duplication at most.